core/storage/user/gae_models.py

```diff
@@ -343,6 +370,11 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_model_association_to_user():
+        """Model is exported as a one instance per user."""
```

Comment:
Should drop "a" here and elsewhere?

Reply:
done

Reply:
I accidentally removed a space here, fixing it now.